src/app/options/time/time.component.html

```diff
@@ -241,6 +241,19 @@ <h4 class="span-2">{{'options.time.binary' | translate}}</h4>
           <label [for]="'binarydim'+i">{{'options.time.binaryDim' | translate}}<tooltip text="{{'options.time.binaryDimDesc' | translate}}"></tooltip></label>
           <options-toggle type="toggle" [name]="'binarydim'+i" [id]="'binarydim'+i" [(ngModel)]="clock.binary.dim"></options-toggle>
         </div>
+        <div class="input" *ngIf="clock.binary.enabled === true">
+          <label [for]="'binaryUseNumbers'+i">{{'options.time.binaryUseNumbers' | translate}}<tooltip text="{{'options.time.binaryUseNumbers' | translate}}"></tooltip></label>
```

Comment:
Rather than repeating the label text, you can just remove the tooltip component here. Only needed if the field needs further explanation. Same for the on and off value too.

Reply:
I gave the binaryOffValue and binaryOnValue also a more descriptive tooltip.

Will you do the other languages or should I also update those files?

Reply:
I'm fine doing the translations later (I just use Google Translate FYI).

Sorry for the confusion, it seems you are still using the same text for the label that displays in the settings and the text that displays in the tooltip. If using the tooltip, these should be different texts. As in, what displays on the page should be a short and sweet label, and the tooltip can clarify with a full sentence if needed. I don't think clarification is needed here though, so I was fine with the text you had before, you just need to remove the tooltip part.